src/WopiValidator/Options.cs

```diff
@@ -20,6 +20,12 @@ class Options
 		[Option('l', "token_ttl", Required = true, HelpText = "WOPI access token ttl")]
 		public long AccessTokenTtl { get; set; }
 
+		[Option('r', "X-WOPI-UsingRestrictedScenario", Required = false, HelpText ="Restricted scenario")]
```

Comment:
`[Option("RestrictedScenario", Required = false, HelpText ="Restricted scenario name; applies only to the GetRestrictedLink and RevokeRestrictedLink operations")]`